configure.py

```diff
@@ -13,7 +13,7 @@
 import bz2
 import io
 
-from distutils.version import StrictVersion
+from packaging.version import Version
```

Comment:
Can you revert this change so the refactor isn't breaking for build tooling and we can review the rest?

Reply:
Sure, I'll do this in a separate PR.